docs/pages/guides/liveblocks-pricing-model.mdx

```diff
@@ -0,0 +1,129 @@
+---
+meta:
+  title: "Liveblocks 1.0 pricing model"
+  parentTitle: "Guides"
+  description: "Explanation of Liveblocks 1.0 pricing model and implementation"
+---
+
+With the release of Liveblocks 1.0, we included new pricing plans that are based
+on monthly active users (MAU). This guide will help you understand how to use
+the new pricing plans by providing guidance on the migration to Liveblocks 1.0
+and how to include a `userId` into the `authorize` option if you use it.
+
+## What is a Monthly Active User (MAU) vs. a connection? [#mau-vs-connection]
+
+In versions before 1.0 we used the number of connections to determine pricing
+segments. We counted each time a user connected to a Liveblocks room from a
+physical device as a connection. While this approximated the number of users, it
+did not account for short-lived connections and was sometimes hard to predict.
+We introduced pricing based on Monthly Active Users (MAU) to reflect the value
+that Liveblocks offers to our customers.
+
+A monthly active user can be defined as:
+
+- Each unique `userId` that connects to a Liveblocks room at least once a month
+  (if using authorization)
+- Each unique cookie that connects to a Liveblocks room at least once a month
+  (if not using authorization)
+
+## How does authorization work with MAU? [#authorization]
+
+In `@liveblocks/node`, we made the field `userId` in the `authorize` option
+mandatory. This `userId` is used to track MAU associated with a Liveblocks
+account. Each `userId` will count as 1 MAU. To make sure you begin to track MAU
+as intended, first update `@liveblocks/node`.
+
+```bash
+npm update @liveblocks/node@1.0
+```
+
+Then, update your `authorize` option to include the `userId` field.
+
+```ts highlight="16,20"
+import { authorize } from "@liveblocks/node";
+
+// Replace this key with your secret key provided at
+// https://liveblocks.io/dashboard/projects/{projectId}/apikeys
+const secret = "sk_prod_xxxxxxxxxxxxxxxxxxxxxxxx";
+
+export default async function auth(req, res) {
+  /**
+   * Implement your own security here.
+   *
+   * It's your responsibility to ensure that the caller of this endpoint
+   * is a valid user by validating the cookies or authentication headers
+   * and that it has access to the requested room.
+   */
+  const room = req.body.room;
+  const response = await authorize({
+    room,
+    secret,
+    // Corresponds to the UserMeta[id] type defined in liveblocks.config.ts
+    userId: "123",
```

Comment:
Why isn't it comment as required like in the example of the 1.0 upgrading guide?